desktop_files/package.json.orig

```diff
@@ -23,24 +23,24 @@
     "node_modules/google-breakpad"
   ],
   "desktopFonts": [
-      "../../../../../resources/fonts/Inter-BlackItalic.otf",
-      "../../../../../resources/fonts/Inter-Black.otf",
-      "../../../../../resources/fonts/Inter-BoldItalic.otf",
-      "../../../../../resources/fonts/Inter-Bold.otf",
-      "../../../../../resources/fonts/Inter-ExtraBoldItalic.otf",
-      "../../../../../resources/fonts/Inter-ExtraBold.otf",
-      "../../../../../resources/fonts/Inter-ExtraLight-BETA.otf",
-      "../../../../../resources/fonts/Inter-ExtraLightItalic-BETA.otf",
-      "../../../../../resources/fonts/Inter-Italic.otf",
-      "../../../../../resources/fonts/Inter-Light-BETA.otf",
-      "../../../../../resources/fonts/Inter-LightItalic-BETA.otf",
-      "../../../../../resources/fonts/Inter-MediumItalic.otf",
-      "../../../../../resources/fonts/Inter-Medium.otf",
-      "../../../../../resources/fonts/Inter-Regular.otf",
-      "../../../../../resources/fonts/Inter-SemiBoldItalic.otf",
-      "../../../../../resources/fonts/Inter-SemiBold.otf",
-      "../../../../../resources/fonts/Inter-Thin-BETA.otf",
-      "../../../../../resources/fonts/Inter-ThinItalic-BETA.otf"
+    "../../../../../resources/fonts/Inter-BlackItalic.otf",
```

Comment:
there's probably no need for this change, and it makes future `git blame` more complicated

Reply:
I added this because some tool that I ran did this automatically, so we'll need to revert this any time we run that tool. Unfortunately I don't recall which tool that was. I'll revert it.